src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -748,6 +749,45 @@
     REASON_TO_FORCE_CASE_TO_SUBMIT_HEARING_REQUIREMENTS(
         "reasonToForceCaseToSubmitHearingRequirements", new TypeReference<String>() {}),
 
+    IS_FEE_PAYMENT_ENABLED(
+            "isFeePaymentEnabled", new TypeReference<YesOrNo>() {}),
+    PAY_FOR_THE_APPEAL_OPTION(
+            "payForTheAppealOption", new TypeReference<String>() {}),
+    PAY_NOW(
+            "payNow", new TypeReference<String>() {}),
+    PAY_LATER(
```

Comment:
Same as comment above, do we in fact need this field if we are writing to the PAY_FOR_THE_APPEAL_OPTION ?

Reply:
Field removed.